.travis.yml

```diff
@@ -169,23 +166,13 @@ jobs:
       python: "3.6"
       env:
         - PYTORCH_CHANNEL=pytorch-nightly
-      if: branch = nightly
```

Comment:
We still need to keep if cron otherwise every PR, push etc there is a Nightly stage : https://travis-ci.org/pytorch/ignite/builds/578850726?utm_source=github_status&utm_medium=notification

It should be something to deal with if TRAVIS_EVENT_TYPE=cron : https://docs.travis-ci.com/user/cron-jobs/#detecting-builds-triggered-by-cron

Maybe something like 
```
env(TRAVIS_EVENT_TYPE) = cron
```

Reply:
Yes, that's why this PR is still a draft, I still have to figure out if a cron job can pass more information apart that it is a cron. Of course if the only cron there is for building nightly this could be enough, but I find it irking :-)

Reply:
I see, maybe we can add another env variable as 
```
      env:
        - NIGHTLY_BUILD=1
```
and check if it is present

Reply:
Well apparently you can't set an environment variable differently in a cron job in Travis yet, so I guess we'll have to make do with just checking the event type indeed :-(